unit_testing/fakes/fake_USBprint.cpp

```diff
@@ -179,7 +179,7 @@ void hostUSBDisconnect()
     connected = false;
 }
 
-void itoa(int n, char* s, int radix)
+void intToA(int n, char* s, int radix)
```

Comment:
I imagine you have renamed the itoa function, because the mingw build imports it already and you get a conflict of definitions. However, this will probably end up braking other unittests (in CA_Embedded/CA_embedded_private) that calls this function. 

So I don't think this is necessarily a change that we'd want to make unless we make sure it doesn't affect other parts of the system.